namespaces/live-1.cloud-platform.service.justice.gov.uk/legal-framework-api-uat/resources/ecr.tf

```diff
@@ -0,0 +1,24 @@
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=4.4"
```

Comment:
Any chance you can bump to latest `4.5`?

https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

Reply:
Will do, although this was auto generated by the cloud-platform CLI.

Reply:
Ah, apologies for that. I'll get the template changed. :+1: